.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.6.6
```

Comment:
This looks like a non-Elixir file. Can you please remove it?

Reply:
Sure. This is the _asdf_ elixir installation client file and it forces the directory to start using a specific elixir version.